Mozilla_Hubs_ToS/en-US.md

```diff
@@ -0,0 +1,78 @@
+# Terms of Service for Hubs and Spoke
+
+Version 3.2, Effective March 12, 2019
```

Comment:
Please add extra information on the release date using this format: https://github.com/mozilla/legal-docs/blob/master/firefox_privacy_notice/en-US.md